content/content-strategy/manage-content-requests/create-content/cards.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: cards/cards
+cards:
+- headline: Download the content production template, 20KB DOCX 
+link: ...
```

Comment:
Ignore style for now: this will be replaced with attachment partial. 
But there's no link to the attachment.

CC @trisha-t and @Phend-117 if we intend to publish this for 23/4 then we'll still need to provide a link to the actual file.

Reply:
@shad0wMe0w Yes, the link is there. It's the tools and templates area that still needs to be resolved - may not be suitable for cards.